x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/constants/url_constants.ts

```diff
@@ -8,10 +8,13 @@
 export enum URL_KEYS {
```

Comment:
so these are not series config keys, i am just using these in a short form to keep url length in check, since right now state is in URL, though we may want to move the state into session storage eventually.